packages/chain-adapters/src/api.ts

```diff
@@ -55,10 +49,4 @@ export type ChainAdapter<T extends ChainTypes> = {
   unsubscribeTxs(input?: chainAdapters.SubscribeTxsInput): void
 
   closeTxs(): void
-} & ChainSpecificAdapterProps<
```

Comment:
This stuff seems unrelated, not sure if there is reason behind this.

Reply:
this will break validating ens address if we remove it i'm pretty sure?

Reply:
We're going to have to figure something out here, then. Changing the ChainAdapter definition from an `interface` to a `type` breaks the templated inheritance model for CosmosSdkBaseAdapter.

```
src/cosmossdk/CosmosSdkBaseAdapter.ts:25:83 - error TS2422: A class can only implement an object type or intersection of object types with statically known members.

25 export abstract class CosmosSdkBaseAdapter<T extends CosmosChainTypes> implements IChainAdapter<T> {
```